src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,77 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int buySum = 0;
+    private int supplySum = 0;
```

Comment:
```suggestion
```
these variables should be local variables

Reply:
I have defined these variables at the class level because I use them in two methods...OK I can try to return an array from calculateSum method...